deploy/package.py

```diff
@@ -17,6 +17,8 @@
 parser.add_argument('--linked_key_vault', type=str, required=True, help="Key Vault to be added as Linked Service")
 parser.add_argument('--location', type=str, required=True, help="Batch Account Location")
 parser.add_argument('--pipeline_name', type=str, required=True, help="Name of the pipeline to package")
+parser.add_argument('--pg_db_username', type=str, required=True, help="Username to login to postgres db")
```

Comment:
suggestion :- it may be good to have these parameters as optional with some default values